### DISCUSSION TITLE
How many sides does a nonagon have?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#422] [Galaxy Trivia] [answered]
**Bonus:** What is a figure with one more side than a nonagon called?

**Comment by siddarthkay** [comment, marked as answer]
A nonagon has 9 sides
A decagon has 10 ( i.e one more than a nonagon)

**Comment by peterbaccaro** [comment]
In geometry, a nonagon is a **nine-sided** polygon or 9-gon.
